modules/ttdBidAdapter.md

```diff
@@ -0,0 +1,127 @@
+# Overview
+
+```
+Module Name: The Trade Desk Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: prebid-maintainers@thetradedesk.com
+```
+
+# Description
+
+Module that connects to The Trade Desk's demand sources to fetch bids.
+
+The Trade Desk bid adapter supports Banner and Video.
+
+# Test Parameters
+
+```js
+    var adUnits = [
+            // Banner adUnit with only required parameters
+            {
+                code: 'test-div-minimal',
+                mediaTypes: {
+                    banner: {
+                        sizes: [[300, 250]]
+                    }
+                },
+                bids: [
+                    {
+                        bidder: 'ttd',
+                        params: {
+                            supplySourceId: 'supplier',
+                            publisherId: '1427ab10f2e448057ed3b422',
+                            siteId: 'site-123',
+                            placementId: 'footer1'
+                        }
+                    }
+                ]
+            },
+            // Banner adUnit with all optional parameters provided
+            {
+                code: 'test-div-banner-optional-params',
+                mediaTypes: {
+                    banner: {
+                        sizes: [[728, 90]],
+                        pos: 1
+                    }
+                },
+                bids: [
+                    {
+                        bidder: 'ttd',
+                        params: {
+                            supplySourceId: 'supplier',
+                            publisherId: '1427ab10f2e448057ed3b422',
+                            siteId: 'site-123',
+                            placementId: 'footer1',
+                            categories: ['IAB23', 'IAB11-3'],
```

Comment:
Please also fix your examples so as to not encourage the bad practice of repeating global setup into bidder params

Reply:
removed